src/Common/RJSF/Form.tsx

```diff
@@ -33,6 +33,7 @@ export const RJSFForm = (props: FormProps) => (
         noHtml5Validate
         showErrorList={false}
         autoComplete="off"
+        className={`rjsf-root-css ${props.className}`}
```

Comment:
Is this a class from rjsf?

Reply:
nope this is a class I introduced for setting the margin right and override the form__input padding

Reply:
improved the name as per suggestion